cli.js

```diff
@@ -142,6 +142,8 @@ process.on('uncaughtException', function(err) {
 
     flags = flags || [];
     optFlags = optFlags || [];
+    // we always take a log option
+    optFlags.push('log');
```

Comment:
We need to have this here, or add it to every `readOptions` call, else you get a `Warn: unknown option log` at the end of every call to a jspm command.